packages/datadog-plugin-mocha/src/index.js

```diff
@@ -156,6 +154,19 @@ class MochaPlugin extends Plugin {
     })
   }
 
+  addMochaSub (channelName, handler) {
+    this.addSub(channelName, (...args) => {
+      if (!this._hasBeenUsed) {
```

Comment:
This could be done in `configure` instead.

Reply:
I don't understand. You mean to include the `_hasBeenUsed` logic in the plugin's `configure` ? `configure` is called  even if the plugin is not used, so it wouldn't work, unless I'm missing something

Reply:
I'm close to finishing a change that will lazy load plugins only when they are needed, which will make this easier to implement. It should be ready next week.